json_serializable/lib/src/encoder_helper.dart

```diff
@@ -160,7 +191,7 @@ abstract class EncodeHelper implements HelperCore {
 
   /// Returns `true` if the field can be written to JSON 'naively' – meaning
   /// we can avoid checking for `null`.
-  bool _writeJsonValueNaive(FieldElement field) {
+  bool _canWriteJsonWithoutNullCheck(FieldElement field) {
```

Comment:
That IS a better name, but in general I'd love these tiny tweaks to be seperate PRs – easier to read things!